server/handlers_v2_test.go

```diff
@@ -2749,3 +2750,436 @@ func TestHTTPServer_GetReportForRequest_NoRedis(t *testing.T) {
 		)
 	}, testTimeout)
 }
+
+func TestHTTPServer_DVONamespaceListEndpoint_NoWorkloads(t *testing.T) {
+	helpers.RunTestWithTimeout(t, func(tt testing.TB) {
+		defer helpers.CleanAfterGock(t)
+
+		err := loadMockRuleContentDir(&testdata.RuleContentDirectory3Rules)
+		assert.Nil(t, err)
+
+		// prepare list of organizations response
+		amsClientMock := helpers.AMSClientWithOrgResults(
+			testdata.OrgID,
+			data.ClusterInfoResult,
+		)
+
+		helpers.GockExpectAPIRequest(
+			t,
+			helpers.DefaultServicesConfig.AggregatorBaseEndpoint,
+			&helpers.APIRequest{
+				Method:       http.MethodGet,
+				Endpoint:     "organization/{organization}/workloads", // TODO: use real ira_server endpoint
```

Comment:
there is a TODO that may have been leftover (?)

Reply:
the aggregator endpoints don't exist, so not a leftover

Reply:
+1